tools/make.py

```diff
@@ -73,8 +73,8 @@
 prefix = "ace"
 pbo_name_prefix = "ace_"
 signature_blacklist = ["ace_server.pbo"]
-importantFiles = ["mod.cpp", "README.md", "AUTHORS.txt", "LICENSE", "logo_ace3_ca.paa"]
-versionFiles = ["README.md", "mod.cpp"]
+importantFiles = ["mod.cpp", "README.md", "documentation\\README_DE.md", "documentation\\README_PL.md", "AUTHORS.txt", "LICENSE", "logo_ace3_ca.paa"]
+versionFiles = ["README.md", "documentation\\README_DE.md", "documentation\\README_PL.md", "mod.cpp"]
```

Comment:
Reminder: need to change location from `documentation` to `docs` when #2377 is merged.